Tests/web3swiftTests/localTests/LocalTestCase.swift

```diff
@@ -15,8 +15,11 @@ class LocalTestCase: XCTestCase {
     override func setUp() async throws {
         let web3 = try! await Web3.new(LocalTestCase.url)
 
-        let block = try! await web3.eth.blockNumber()
-        if block >= 25 { return }
+        guard let block = try? await web3.eth.blockNumber() else {
+            return
+        }
+
+        guard block < 25 else { return }
```

Comment:
This should throw though, right? `try!` was probably done on purpose here.

The setup would not fail this way any longer if a block number could not be fetched?

Reply:
Agreed with this one, this function should not failing silently, since it now actually very helpful to get that you've forgot to start ganache)

Reply:
addressed in next commit